lib/v2/nocturnalscans/chapters.js

```diff
@@ -0,0 +1,55 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+
+module.exports = async (ctx) => {
+    const {name} = ctx.params;
+    const url = `https://nocturnalscans.com/MANGA/${name}/`;
+    const response = await got({
+        method: 'get',
+        url,
+    });
```

Comment:
```suggestion
    const response = await got(url);
```
got request can be simplified